data/Music/Metroid Series/Metroid Prime 3/Rundas Battle.meta

```diff
@@ -0,0 +1,3 @@
+Metroid Prime 3: Corruption - Rundas Battle
+0x22
+Battle,MinibossBattle,BossBattle,FireBoss,GanondorfBattle,GanonBattle,SmallFight,BossFight,BigFight,Fight,FInalFight
```

Comment:
```suggestion
Battle,MinibossBattle,BossBattle,FireBoss,GanondorfBattle,GanonBattle,SmallFight,BossFight,BigFight,Fight,FinalFight
```

Reply:
I swore I fixed that, I must not've saved the file afterwards...whoops.